docs/Validation.md

```diff
@@ -282,7 +282,7 @@ const emailValidators = [required(), validateEmailUnicity];
 
 export const UserCreate = () => (
     <Create>
-        <SimpleForm validate={validateUserCreation}>
```

Comment:
The `validateUserCreation` does not exist in this example

Reply:
??

Reply:
nvm